README.md

```diff
@@ -34,27 +34,14 @@
 <!-- About the Project -->
 ## :star2: About the Project
 
-InvenTree is an open-source Inventory Management System which provides powerful low-level stock control and part tracking. The core of the InvenTree system is a Python/Django database backend which provides an admin interface (web-based) and a REST API for interaction with external interfaces and applications.
+InvenTree is an open-source Inventory Management System which provides powerful low-level stock control and part tracking. The core of the InvenTree system is a Python/Django database backend which provides an admin interface (web-based) and a REST API for interaction with external interfaces and applications. A powerful plugin system provides support for custom applications and extensions.
 
-InvenTree is designed to be lightweight and easy to use for SME or hobbyist applications, where many existing stock management solutions are bloated and cumbersome to use. Updating stock is a single-action process and does not require a complex system of work orders or stock transactions.
-
-Powerful business logic works in the background to ensure that stock tracking history is maintained, and users have ready access to stock level information.
-
-<!-- Features -->
-
-### :dart: Features
-
-- Organize Parts and BOMs
-- Manage Suppliers
-- Instant Stock Knowledge
-- Extend and Customize to fit your usage
+Check out [our website](https://inventree.org) for more details.
 
 <!-- Roadmap -->
 ### :compass: Roadmap
 
-* [x] Plugins
-* [ ] Improved Importers
-* [ ] Custom Workflow Processes
```

Comment:
@SchrodingersGat from my experience a lot of people only look at the readme. Having some items there is probably beneficial to garner interest.